lib/src/config/constants.dart

```diff
@@ -1,7 +1,19 @@
 class AppConstants {
-  static const String ENV = 'ENV';
-  static const String HOST_NAME = 'HOST_NAME';
-  static const String DEFAULT_ENV = 'production';
-  static const String ENV_CONFIG_PATH = 'assets/config/env/';
-  static const String ENV_CONFIG_TYPE = '.json';
+  static const String env = 'ENV';
+  static const String hostName = 'HOST_NAME';
+  static const String coinPath = 'COIN_PATH';
+  static const String marketPath = 'MARKET_PATH';
+  static const String currency = 'CURRENCY';
+  static const String perPage = 'PER_PAGE';
+  static const String page = 'PAGE';
+  static const String sparkline = 'SPARKLINE';
+  static const String order = 'ORDER';
+  static const String defaultEnv = 'production';
+  static const String envConfigPath = 'assets/config/env/';
+  static const String envConfigType = '.json';
+  static const String currencyOfMarket = 'usd';
+  static const String orderBy = 'market_cap_desc';
+  static const String amountPerPage = '20';
+  static const String numberPage = '1';
+  static const String valueSparkline = 'false';
```

Comment:
boolean variables should start with `is`

Reply:
I will change it